### PR TITLE
nanodbc: add nanodbc/cci.20200807 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **nanodbc/cci.20200807**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
Require c++11, you can force CXX_STANDARD.

**Comment by madebr** [comment]
> Require c++11, you can force CXX_STANDARD.

It's c++14, I have to exclude older compilers.

File: recipes/nanodbc/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.14)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+if(WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```
Comment (danimtb) [inline recipes/nanodbc/all/CMakeLists.txt:8]: this requires CMake 3.4 at least

File: recipes/nanodbc/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.14)
```
Comment (jgsogo) [inline recipes/nanodbc/all/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.4)
```

Check comment by @danimtb  https://github.com/conan-io/conan-center-index/pull/2460/files#r470514373

File: recipes/nanodbc/all/CMakeLists.txt (hunk 1 above)
Comment (madebr) [reply recipes/nanodbc/all/CMakeLists.txt:8]: Thanks. I bumped the required version.

**Pull request merged**